therapy/etl/chembl.py

```diff
@@ -16,46 +16,34 @@
 class ChEMBL(Base):
     """ETL the ChEMBL source into therapy.db."""
 
-    def _extract_data(self, *args, **kwargs):
+    def _extract_data(self):
         """Extract data from the ChEMBL source."""
-        logger.info('Extracting chembl_27.db...')
-        if 'data_path' in kwargs:
-            chembl_db = kwargs['data_path']
+        self._src_data_dir.mkdir(exist_ok=True, parents=True)
+        chembl_files = list(self._src_data_dir.glob('chembl_*.db'))
+        if chembl_files:
```

Comment:
This does not get the latest chembl version if previous versions are already present

Reply:
Yeah, is that intended behavior? I was thinking about doing an across-the-board `--force-update` option to fetch the latest version of everything if we want to make that available.

Reply:
Yes, that was the reason for this issue. I think it makes sense for the default behavior to fetch the latest version. If they didn't want to update a source, they would just exclude it from the `normalizer` flag in CLI

Reply:
Hm... for reproduceability purposes, we may want to be able to load from older files somehow, but maybe we can cross that bridge later.

Reply:
Ya we have this old [issue](https://github.com/cancervariants/gene-normalization/issues/15) in gene-normalizer regarding this